### ISSUE TITLE
PCA con ggrepel

### ISSUE THREAD

**Issue by APonce73** [ComunidadBioInfo/cdsb2019#8]
Existen varios paquetes para hacer Análisis de componentes principales (PCA), como *vegan* o *ade4*. Sin embargo su visualización no es muy atractiva.

con ggplot se han hecho muchas visualizaciones y existe un paquete llamado
*ggord* donde visualiza de forma amigable los PCA. Sin embargo no incluye la función #repel del paquete *ggrepel*. Por lo que se hizo esta función

https://github.com/APonce73/PCARepel

**Comment by lcolladotor** [comment]
¿Entonces la idea sería completar el paquete PCARepel verdad?

**Comment by APonce73** [comment]
Exacto... ya tenía un código y una función hace tiempo y sería combertirla como parte de un paquete nuevo :)

**Comment by lcolladotor** [comment]
va va , suena muy bien =)
